samples/cellular/nrf_cloud_rest_cell_location/src/main.c

```diff
@@ -367,52 +360,10 @@ static void do_jitp(void)
 }
 #endif
 
-#define REQ_CARD "---> Press button %d to enable the location card for this device on nrfcloud.com"
-static void request_location_card_enable(void)
-{
-	if (!location_card_enable) {
-		int err;
-
-		k_sem_reset(&button_press_sem);
-
-		LOG_INF(REQ_CARD, BTN_NUM);
-		LOG_INF("     This only needs to be done once per device");
-		LOG_INF("     Waiting %d seconds...", UI_REQ_WAIT_SEC);
-
-		err = k_sem_take(&button_press_sem, K_SECONDS(UI_REQ_WAIT_SEC));
-
-		if (err == 0) {
-			location_card_enable = true;
-		} else {
-			if (err != -EAGAIN) {
-				LOG_ERR("k_sem_take error: %d", err);
-			}
-		}
-	}
-
-	if (location_card_enable) {
-		LOG_INF("Location card will be enabled after network connection is obtained");
-	} else {
-		LOG_INF("Location card will not be enabled");
-	}
-}
-
-static void do_location_card_enable(void)
+static void send_device_status(void)
 {
 	int err;
 
-	struct nrf_cloud_svc_info_ui ui = {
-		.gnss = 1,
-		.log = IS_ENABLED(CONFIG_NRF_CLOUD_LOG_BACKEND) &&
-		       IS_ENABLED(CONFIG_LOG_BACKEND_NRF_CLOUD_OUTPUT_TEXT),
-		.dictionary_log = IS_ENABLED(CONFIG_NRF_CLOUD_LOG_BACKEND) &&
```

Comment:
We need to verify that binary vs. text logging is automatically rendered correctly now.